app/controllers/twitter.js

```diff
@@ -1,23 +1,213 @@
+		"profile_background_tile": false,
+		"profile_link_color": "1B95E0",
+		"profile_sidebar_border_color": "000000",
+		"profile_sidebar_fill_color": "000000",
+		"profile_text_color": "000000",
+		"profile_use_background_image": false,
+		"profile_image_url": "http://pbs.twimg.com/profile_images/758871468428013568/nbZ4gwPc_normal.jpg",
+		"profile_image_url_https": "https://pbs.twimg.com/profile_images/758871468428013568/nbZ4gwPc_normal.jpg",
+		"profile_banner_url": "https://pbs.twimg.com/profile_banners/758842850511495168/1469764834",
+		"default_profile": false,
+		"default_profile_image": false,
+		"following": null,
+		"follow_request_sent": null,
+		"notifications": null
+	},
+	"geo": {
+		"type": "Point",
+		"coordinates": [40.768927, -111.808184]
+	},
+	"coordinates": {
+		"type": "Point",
+		"coordinates": [-111.808184, 40.768927]
+	},
+	"place": {
+		"id": "1879ace9e02ace61",
+		"url": "https://api.twitter.com/1.1/geo/id/1879ace9e02ace61.json",
+		"place_type": "admin",
+		"name": "Utah",
+		"full_name": "Utah, USA",
+		"country_code": "US",
+		"country": "United States",
+		"bounding_box": {
+			"type": "Polygon",
+			"coordinates": [
+				[
+					[-114.052999, 36.997905],
+					[-114.052999, 42.001619],
+					[-109.041059, 42.001619],
+					[-109.041059, 36.997905]
+				]
+			]
+		},
+		"attributes": {}
+	},
+	"contributors": null,
+	"is_quote_status": false,
+	"retweet_count": 0,
+	"favorite_count": 0,
+	"entities": {
+		"hashtags": [{
+			"text": "SLC",
+			"indices": [74, 78]
+		}, {
+			"text": "PokemonGO",
+			"indices": [79, 89]
+		}],
+		"urls": [{
+			"url": "https://t.co/JWpOPLfVpA",
+			"expanded_url": "https://maps.google.com/maps?q=40.768927,-111.808184",
+			"display_url": "maps.google.com/maps?q=40.7689…",
+			"indices": [90, 113]
+		}],
+		"user_mentions": [],
+		"symbols": []
+	},
+	"favorited": false,
+	"retweeted": false,
+	"possibly_sensitive": false,
+	"filter_level": "low",
+	"lang": "en",
+	"timestamp_ms": "1471551440854"
+}
+
+important fields: 
+	create_at	: UTC time when this Tweet was created.
+	coordinates	: he longitude and latitude of the Tweet’s location.
+	text		: The actual UTF-8 text of the status update.T.
 
-let Writable = require('stream').Writable,
+*/
+
+let fs = require('fs'),
+    config = require('../../config'),
+	Writable = require('stream').Writable,
     TwitterStream = require('twitter-stream-api'),
-    config = require('../../config');
+	pokemonTweet = require(__appbase + 'models/pokemonAppearances.js'),
+    pokemonListPath = require(__appbase + '../resources/json/pokemonIdAndName.json');
 
 const Twitter = new TwitterStream(config.twitter);
 
+/*
+	keywords used to search for in the tweets
+	,  : represents OR
+	' ': space represents AND 
+*/
+const tweetSearchKeywords = 'pokemon catch,pokemon saw,pokemon attack,pokemon find,pokemon caught,pokemon attacked,pokemon found,pokemon appeared,#foundPokemon,#caughtPokemon,#pokemongo,a wild appeared until #pokemongo';
```

Comment:
You might want to define these somewhere else, eg. in the config file. Best would be to have a set of predefined keywords (as the ones here)  in the config file :).